component/loki/source/windowsevent/component_windows.go

```diff
@@ -152,5 +152,14 @@ func convertConfig(arg Arguments) *scrapeconfig.WindowsEventsTargetConfig {
 		ExcludeEventData:     arg.ExcludeEventData,
 		ExcludeEventMessage:  false,
 		ExcludeUserData:      arg.ExcludeUserdata,
+		Labels:               toLabelSet(arg.Labels),
 	}
 }
+
+func toLabelSet(in map[string]string) model.LabelSet {
```

Comment:
this function is also used in loki.write. I guess that it could be put in some loki generic types file to avoid the duplication. There is a common/loki/types.go but it's a copy from Promtail so it should maybe not be modified.